tests/lib.rs

```diff
@@ -652,6 +652,35 @@ fn test_multiple_platform_command_search_not_found() {
         .stderr(contains("Page `windows-only` not found in cache."));
 }
 
+#[test]
+fn test_osx_is_alias_for_macos() {
```

Comment:
I think we usually phrase it the other way around:
```suggestion
fn test_macos_is_alias_for_osx() {
```